src/main/java/be/xplore/pricescraper/domain/shopping/Item.java

```diff
@@ -0,0 +1,16 @@
+package be.xplore.pricescraper.domain.shopping;
+
+import jakarta.persistence.*;
```

Comment:
Star-imports aren't recommended. Make sure your java code-style settings in your IDE automatically optimize imports.